lib/services/signalrManagement/package.json

```diff
@@ -0,0 +1,25 @@
+{
+  "name": "azure-arm-signalr",
+  "author": "Microsoft Corporation",
+  "description": "SignalRManagementClient Library with typescript type definitions for node",
+  "version": "1.0.0",
```

Comment:
1.0.0-preview

Reply:
Yup. I'll regenerate as soon as my azure-rest-api-specs PR is merged.

Reply:
Done.